staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/utils.go

```diff
@@ -20,9 +20,9 @@ import (
 	"fmt"
 	"path/filepath"
 	"regexp"
+	"strconv"
 	"strings"
 
-	"github.com/blang/semver"
```

Comment:
How is the new parsing + comparison different to `blang/semver`? It seems to me (at least in the unit tests) that semver should handle it too.

Reply:
`blang/semver` does not support 4 level version `1.2.3.4`.

Reply:
thanks for the info